_episodes/|1-06-issues.md

```diff
@@ -0,0 +1,105 @@
+- "Task lists can be created to keep track of a list of issues"
+- "Formatting syntaxes, templates, and subscribing to issues help with communication"
+---
+
+## GitHub issues
+
+- GitHub issues are a feature of GitHub. They let you keep track of your work on a GitHub repository and are used to report bugs, request features or enhancements, or to discuss implementation details of some parts of the code.
+ - Issues are a kind of TODO list, with pending and completed tasks as well as serving to prioritise the development activity.
+ - Labels can be added to the issues by the repository administrator to inform at a first glance what the issue is about. Typical labels are “bug”, “enhancement”, “low priority” or “good first issue”, for example.
+ - Issues also can have one or more people assigned to them who will take care of sorting them out and closing them when done or if no longer relevant.
+ - By default any GitHub user can create an issue in a public repository
+
+The following figure shows some of the issues open in a certain repository. The labels tell us there are a couple of bug reports, a couple of issues related to the performance of the software and several ones that are simple enough to be tackled by novice people. Most of them have some discussion going on.
+
+ ![Git collaborative]({{ site.baseurl }}/fig/issues1.png "Example of issues."){:class="img -responsive"}
+
+## Creating GitHub issues
+It is possible to create issues in a number of ways (see the GitHub docs page on [creating issues][github-creating-issues] to learn about the various methods). One of the most common and simplest methods is to create an issue from a repository. This can be done by navigating to the main page of the repository on GitHub and click on `Issues`, then click on the `New Issue` button.
+
+> ## Create two new issues
+> Follow the information above to create two new issues in your example repository. It doesn't matter what the issue title or contents are at this stage while we are learning how to create issues, but generally this is an important, and you should always refer to a repository's contributing guidelines for how to construct your issue. For now some simple example titles such as "Fix bug" and "Create Feature" will suffice. Try adding some labels to your issues and assigning yourself.
+>> ## Solution
+>> You have created two new issues and your issues page should look something like this:
+>> ![New Issues](../fig/issues2.png)
+>> Notice the labels have been added next to the issue title, and the assignees' profile pictures are visible too.
+> {: .solution}
+{: .challenge}
+
+[github-creating-issues]: https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue
+
+> ## Mentioning other issues
+> All issues receive a tag number starting at 1 and preceded by #, 
+> like #40 or #110. If you want to refer to an issue in any comment
+> anywhere in GitHub, just use its tag number and these will be automatically
```

Comment:
This is true for issues within the repo. If you want to mention an issue in another repo, you need to indicate what repo you're talking about. Typically, GitHub will do this for you nicely if you just paste the URL of the issue you want to mention.